zerver/tests/test_subs.py

```diff
@@ -2207,25 +2207,6 @@ def test_change_stream_message_retention_days(self) -> None:
         )
         self.assert_json_error(result, "Bad value for 'message_retention_days': 0")
 
-    def test_change_stream_message_retention_days_requires_realm_owner(self) -> None:
-        user_profile = self.example_user("iago")
-        self.login_user(user_profile)
-        realm = user_profile.realm
-        stream = self.subscribe(user_profile, "stream_name1")
-
-        result = self.client_patch(
-            f"/json/streams/{stream.id}", {"message_retention_days": orjson.dumps(2).decode()}
-        )
-        self.assert_json_error(result, "Must be an organization owner")
-
-        do_change_user_role(user_profile, UserProfile.ROLE_REALM_OWNER, acting_user=None)
-        result = self.client_patch(
-            f"/json/streams/{stream.id}", {"message_retention_days": orjson.dumps(2).decode()}
-        )
-        self.assert_json_success(result)
-        stream = get_stream("stream_name1", realm)
-        self.assertEqual(stream.message_retention_days, 2)
```

Comment:
I'm not sure moving this actually improves how tests are grouped.

Reply:
Hmm, yes the permission to change this setting is not controlled by `can_administer_channel_group`, but still false on under testing permissions on updated channels.